data/mods/Magiclysm/Spells/kelvinist.json

```diff
@@ -397,5 +397,46 @@
     "energy_source": "MANA",
     "difficulty": 5,
     "damage_type": "heat"
+  },
+  {
+    "id": "freezing_touch_slow_effect",
+    "type": "SPELL",
+    "name": "Freezing Touch",
+    "description": "This is a sub-spell of Freezing touch.",
+    "shape": "blast",
+    "effect": "attack",
+    "effect_str": "slow_freeze_effect",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SILENT" ],
+    "min_duration": 1000,
+    "max_duration": 1000,
+    "min_range": 1,
+    "max_range": 1
+  },
+  {
+    "id": "freezing_touch",
+    "type": "SPELL",
+    "name": "Freezing Touch",
+    "description": "Your hands freeze anything they touch at temperatures so cold it slows down your foes.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SOMATIC", "NO_PROJECTILE" ],
+    "components": "spell_components_freezing_touch",
+    "effect": "attack",
+    "effect_str": "cold",
```

Comment:
what's this?

Reply:
umm sorry, I don't know what you mean if you mean the components I was trying to make a spell that used items like you wanted. but I'm going to take a guess and say I did something wrong I'm not seeing.

Reply:
"effect_str": "cold"
is the line this comment is pointing at.

Reply:
> "effect_str": "cold"
> is the line this comment is pointing at.

It's the same in chilling touch.

Reply:
ah yes, carry on.